### PR TITLE
feat : add bangkok meetup

### PR DESCRIPTION
**Pull request by houssembaltii**
Image url :https://unsplash.com/fr/photos/temple-qeibBI7w4fQ

### CODE REVIEW

File: src/content/meetup/2024-09-27-bangkok/about.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                    </span>
+                  </li>
+                )}
+              </ul>
+              <p>Description for the location&apos;s meetup</p>
+              <Button variant="default" asChild className="w-fit">
+                <Link href="" target="_blank">
+                  Visit Location Website
+                </Link>
+              </Button>
```
Comment (yoannfleurydev) [inline src/content/meetup/2024-09-27-bangkok/about.tsx:74]: Remove if href is empty

File: src/content/meetup/2024-09-27-bangkok/about.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    <div id="venue" className="relative bg-gray-900">
+      <div className="mx-auto max-w-7xl lg:flex lg:justify-between lg:px-8 xl:justify-end">
+        <div className="flex-col lg:flex lg:w-1/2 lg:shrink lg:grow-0 xl:absolute xl:inset-y-0 xl:right-1/2 xl:w-1/2">
+          <div className="relative h-80 lg:-ml-8 lg:h-auto lg:w-full lg:grow xl:ml-0">
+            <Image
+              className="absolute inset-0 h-full w-full bg-gray-950 object-cover object-left"
+              src={LocationImage}
+              width={1000}
+              height={500}
+              alt=""
```
Comment (yoannfleurydev) [inline src/content/meetup/2024-09-27-bangkok/about.tsx:24]: Need an alt on the image

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]

File: src/content/meetup/2024-09-27-bangkok/about.tsx (hunk 1 above)
Comment (houssembaltii) [reply src/content/meetup/2024-09-27-bangkok/about.tsx:74]: i have asked rudy if we have info about where the meetup will be held , i ll update it when i get the info , if there is none i will delete

File: src/content/meetup/2024-09-27-bangkok/about.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                      aria-hidden="true"
+                    />
+                    <span>
+                      <time
+                        dateTime={props.meetup.date}
+                        className="font-semibold text-primary"
+                      >
+                        {formatDateTime(props.meetup.date)}.
+                      </time>{" "}
+                      Come enjoy conferences and a networking cocktail .
```
Comment (JeanneGrenet) [inline src/content/meetup/2024-09-27-bangkok/about.tsx:64]: cocktail.

**Review by yoannfleurydev — APPROVED** [review]

**Pull request merged**